src/sentry/static/sentry/app/components/alerts.jsx

```diff
@@ -20,8 +20,8 @@ const Alerts = React.createClass({
   render() {
     return (
       <div {...this.props}>
-        {this.state.alerts.map(function(alert, key) {
-           return <AlertMessage id={alert.id} key={key} type={alert.type} message={alert.message} url={alert.url} />;
```

Comment:
I don't think this `key` behavior ever worked correctly, so I explicitly added it as a property on `alert`. Feels like a hack.